src/components/user-tools/author-tools.tsx

```diff
@@ -382,7 +380,7 @@ export function AuthorTools({ tools, entityId, data }: AuthorToolsProps) {
             }
 
             setTimeout(() => {
-              refreshFromAPI()
+              csrReload()
```

Comment:
That looks like a hack. Don't we know the state after the GraphQL query anyway and can update that w/o forcing a refresh?

Reply:
Follow up in https://github.com/serlo/frontend/issues/851